libs/external-router/driver/magento/src/model/resolution-types.ts

```diff
@@ -0,0 +1,5 @@
+export enum MagentoUrlRewriteEntityTypeEnum {
```

Comment:
What's the purpose of this enum? How is it meant to be used? I would add some docs to this.

Reply:
I don't really know. This is just the base type that v2.4.2 gives us. I wasn't 100% sure whether or not we also try to unify across something like this or not.